testing/test_assertrewrite.py

```diff
@@ -1868,16 +1867,13 @@ def test_simple():
             id="assert statement not on own line",
         ),
         pytest.param(
-            b"def x():\n"
```

Comment:
Oups, I missed this, I only searched for ``" f"`` and ``" "``, after fixing it would be ``b"def x():\n    assert (\n        1 + 2  # comment\n    )\n",`` not extraordinary, but better I guess ?

Reply:
Another way would be to change to:
```python
b"""def x():
    assert (
        1 + 2  # comment
    )
"""
```
But I don't know if it works for bytes.
```

Reply:
Multiline b works but I think it will be annoying with the indentation.

Maybe just put `# fmt: off`/`# fmt: on` around them?

Reply:
The scope of ``# fmt: off/# fmt: on`` might not be exactly the same in ruff vs black, I had to "un-nest" the  existing ``fmt: off``.

Reply:
[3d53bbe](https://github.com/pytest-dev/pytest/pull/11901/commits/3d53bbebec39f0c7839fff747f0e5ac6a4dd546b)